packages/babel-core/test/fixtures/transformation/source-maps/input-source-map-complex/output.js

```diff
@@ -88,15 +84,21 @@ exports.default = void 0;
 //
 //
 //
+
 //
```

Comment:
This could indicate a generator bug, since the input also contains this blank line, which was not printed correctly before, but that's not important.